src/test/resources/update+insert/NEW_Lambda.java

```diff
@@ -0,0 +1,7 @@
+import java.util.function.BiFunction;

```

Comment:
```java
import java.util.function.Function;
public class Lambda {
    public static void main(String[] args) {
        BiFunction<String, String, String> lambda = ( x, y) -> {
            return y + " ";
        };
    }
}
```

The pretty-printing of `CtLambda` is quite weird. It prepends an extra space.

Reply:
That is indeed strange. I'll have a look at that.